### PR TITLE
fix 'Script error for http://www.sahibinden.com/ after proxing' (close #644)

### LINKED ISSUE #644: Script error for http://www.sahibinden.com/ after proxing
See https://testcafe-hhhm.devexpress.com/history?timestamp=1467140702957&siteUrl=http://www.sahibinden.com.
Reproduced in playground.

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

**Comment by miherlosev** [comment]
lgtm.
@VasilyStrelyaev check test name

**Comment by churkin** [comment]
lgtm

File: test/client/fixtures/utils/url-test.js [hunk 1]
```diff
@@ -470,3 +470,25 @@ test('"base" tag with an href attribute that is set to a protocol relative url (
 
     base.parentNode.removeChild(base);
 });
+
+test('resolving url writing with the "base" tag (GH-644)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/utils/url-test.js]:
```
test('resolving a url written with the "base" tag (GH-644)', function () {
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by LavrovArtem** [comment]
FPR

**Comment by LavrovArtem** [comment]
ping @VasilyStrelyaev

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**